files/fr/mdn/writing_guidelines/howto/document_a_css_property/index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Comment documenter une propriété CSS
+slug: MDN/Writing_guidelines/Howto/Document_a_CSS_property
+---
+
+{{MDNSidebar}}
+
+Au fur et à mesure que les normes [CSS](/fr/docs/Web/CSS) évoluent, de nouvelles propriétés sont ajoutées. La [référence CSS](/fr/docs/Web/CSS/Reference) sur MDN Web Docs doit être maintenue à jour avec ces développements. Cet article fournit des instructions pas à pas pour créer une page de référence des propriétés CSS.
+
+Chaque page de référence des propriétés CSS suit la même structure. Cela permet aux lecteurs de trouver plus facilement les informations, surtout lorsqu'ils sont familiarisés avec le format standard des pages de référence.
```

Comment:
formulation épicène
```suggestion
Chaque page de référence des propriétés CSS suit la même structure. Cela permet au lectorat de trouver plus facilement les informations, surtout lorsqu'il est familiarisé avec le format standard des pages de référence.
```

Reply:
C'est noté, je serai plus attentif lors de mes prochaines traductions à utiliser au maximum une formulation épicène.